README.md

```diff
@@ -1,37 +1,119 @@
+# bt-web
+Biztech's web application.
+
+## Getting Started
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. See [deployment](#deployment) for notes on how to deploy the project to a live system.
+
+### Installation and Running
+
+1. Clone the repo:
+
+    ```
+    $ git clone https://github.com/ubc-biztech/bt-web
+    ```
+
+2. Install the packages:
+
+    ```
+    $ npm install
+    ```
+
+3. Your setup is ready for development! Run `npm start` to run your app on your local machine.
+
+
+## Development
+
+### Files and Services
+
+* `components/` - Our main view/display components
+* `pages/` - Our main pages, where most of our app logic is handled
+
+* `assets/` - Images, fonts, vectors, and other static files
+* `constants/` - Abstracted constants
+* `utils/` - Any other miscellaneous functions
```

Comment:
Functions to call our backend API, AWS services, and any other miscellaneous functions